sdks/python/apache_beam/runners/interactive/display/pcoll_visualization.py

```diff
@@ -0,0 +1,258 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Module visualizes PCollection data.
+
+For internal use only; no backwards-compatibility guarantees.
+Only works with Python 3.5+.
+"""
+from __future__ import absolute_import
+
+import base64
+import logging
+from datetime import timedelta
+
+from pandas.io.json import json_normalize
+
+from apache_beam import pvalue
+from apache_beam.runners.interactive import interactive_environment as ie
+from apache_beam.runners.interactive import pipeline_instrument as instr
+from facets_overview.generic_feature_statistics_generator import GenericFeatureStatisticsGenerator
+from IPython.core.display import HTML
+from IPython.core.display import Javascript
+from IPython.core.display import display
+from IPython.core.display import display_javascript
+from IPython.core.display import update_display
+from timeloop import Timeloop
+
+# jsons doesn't support < Python 3.5. Work around with json for legacy tests.
```

Comment:
Why do we need this workaround if we only support Python 3.5+?

Reply:
All the Gradle tasks still supports py2.
So you still have docs, lint and tasks running in py2 virtual env. We'll have to pass those pre-commit checks.

Reply:
Can we just print a warning and exit, or exclude this from running in py2? We probably don't want this to create an impression that this will work with py2.

Reply:
This will still work Py2 though. And a sys.exit() would probably break some gradle tasks waiting for 0 exit code.
I've put a TODO item here. It's a general ticket for Interactive Beam to mark for cleanup once py2 is deprecated completely from Beam.

Reply:
Can we at least print a warning that says "interactive is not supported in py2", like we do for python 3 with beam <= 2.15?

Reply:
I've added such warning message in the interactive_environment module to check for Python version and print out warnings.